pgml-cms/docs/introduction/getting-started/README.md

```diff
@@ -4,11 +4,11 @@ description: Setup a database and connect your application to PostgresML
 
 # Getting Started
 
-A PostgresML deployment consists of multiple components working in concert to provide a complete Machine Learning platform. We provide a fully managed solution in our cloud.
+A PostgresML deployment consists of multiple components working in concert to provide a complete Machine Learning platform. We provide a fully managed solution in our cloud, and document a self-hosted installation in our docs.
```

Comment:
documenting the self hosted version is confusing here. Are these docs for cloud users, or self hosted people?

Reply:
Both.